addGradientInfill.py

```diff
@@ -1,49 +1,55 @@
-##################################################
-## Gradient Infill for 3D prints
-##################################################
-## MIT license
-##################################################
-## Author: Stefan Hermann - CNC Kitchen
-## Version: 1.0
-##################################################
+#!/usr/bin/env python3
```

Comment:
You can do this with every script. The `env` tool than finds the proper interpreter no matter where it is installed.